ConsoleApp2/Menu.cs

```diff
@@ -0,0 +1,112 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ConsoleApp2
+{
+    public class Menu
+    {
+        public static void PrintMenu(MenuOptions selectedOption)
```

Comment:
If a method implies that it will be used only within the class, you should mark it as private. Check all the methods you have. Read how methods in a class should be organised if they have different access modifiers.